sdk/@launchdarkly/observability-android/lib/src/main/kotlin/com/launchdarkly/observability/client/InstrumentationManager.kt

```diff
@@ -34,21 +38,25 @@ class InstrumentationManager(
     private val application: Application,
     private val sdkKey: String,
     private val resources: Resource,
+    private val logger: LDLogger,
+    options: Options,
 ) {
     private val otelRUM: OpenTelemetryRum
     private var otelMeter: Meter
     private var otelLogger: Logger
     private var otelTracer: Tracer
 
     init {
-
-        otelRUM = OpenTelemetryRum.builder(application)
+        val otelRumConfig = OtelRumConfig().setSessionConfig(
+            SessionConfig(backgroundInactivityTimeout = options.sessionBackgroundTimeout)
+        )
+        otelRUM = OpenTelemetryRum.builder(application, otelRumConfig)
             .addLoggerProviderCustomizer { sdkLoggerProviderBuilder, application ->
                 val logExporter = OtlpHttpLogRecordExporter.builder()
-                    .setEndpoint(URL_LOGS)
+                    .setEndpoint(options.otlpEndpoint + LOGS_PATH)
+                    .setHeaders { options.customHeaders }
```

Comment:
what's the value in supporting the headers? is it to authenticate the data to a project (in addition to us sending the `highlight.project_id` resource attribute)?